trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/_macros/section-exporter/exporter-table.njk

```diff
@@ -1,100 +1,162 @@
+      } %}
+    {% else %}
+      {% set creditRatingRow = {
+        key: { html: exporterCreditRatingHeadingHtml },
+        value: { html: exporterCreditRatingValueHtml }
+      } %}
+    {% endif %}
+    
+  {% else %}
+
+    {% set exporterCreditRatingValueHtml %}
+      {{ govukTag({
+        text: 'Not added',
+        classes: 'govuk-tag--yellow',
+        attributes: {
+          'data-cy': 'exporter-table-credit-rating-not-added'
+        }
+      }) }}
+    {% endset %}
+
+    {% if userCanEditGeneral %}
+      {% set creditRatingRow = {
+      key: { html: exporterCreditRatingHeadingHtml },
+      value: { html: exporterCreditRatingValueHtml },
+        actions: {
+          items: [
+            {
               href: "/case/" + caseId + "/underwriting/pricing-and-risk/edit",
+              text: "Add",
               attributes: {
                 "data-cy":"add-credit-rating-link"
               }
-            }) }}
-          {% endif %}
-        </td>
-
-      </tr>
-
-      <tr class="govuk-table__row">
-
-        <td scope="row" class="govuk-table__header govuk-body-s govuk-table__cell width-half" data-cy="exporter-table-loss-given-default-heading">
-          Loss given default
-        </td>
-
-        <td class="govuk-table__cell" data-cy="exporter-table-loss-given-default-value">
-          {% if lossGivenDefault %}
-            {{ lossGivenDefault }}%
-          {% else %}
-            -
-          {% endif %}
-        </td>
-
-        <td class="govuk-table__cell govuk-table__cell--border-top text-align-right">
-          {% if userCanEditGeneral %}
-            <a class="govuk-link" href="/case/{{ caseId }}/underwriting/pricing-and-risk/loss-given-default" data-cy="exporter-table-change-loss-given-default-link">Change</a>
-          {% endif %}
-        </td>
-
-      </tr>
-
-      <tr class="govuk-table__row">
-
-        <td scope="row" class="govuk-table__header govuk-body-s govuk-table__cell width-half" data-cy="exporter-table-probability-of-default-heading">
-          Probability of default
-        </td>
-
-        <td class="govuk-table__cell" data-cy="exporter-table-probability-of-default-value">
-
-          {% if probabilityOfDefault %}
-            Less than {{ probabilityOfDefault }}%
-
-            {% else %}
-            -
-          {% endif %}
-        </td>
-
-        <td class="govuk-table__cell govuk-table__cell--border-top text-align-right">
-          {% if userCanEditGeneral %}
-            <a class="govuk-link" href="/case/{{ caseId }}/underwriting/pricing-and-risk/probability-of-default" data-cy="exporter-table-change-probability-of-default-link">Change</a>
-          {% endif %}
-        </td>
-      </tr>
-
-    </tbody>
-  </table>
+            }
+          ]
+        }
+      } %}
+    {% else %}
+      {% set creditRatingRow = {
+        key: { html: exporterCreditRatingHeadingHtml },
+        value: { html: exporterCreditRatingValueHtml }
+      } %}
+    {% endif %}
+
+  {% endif %}
```

Comment:
I had some trouble reading these nested ifs. I think separating the nesting like below would make it more readable. The reason I think this is more readable (other than less nesting generally being easier to read) is that it's (hopefully) more apparent that `exporterCreditRating` determines what text we show and `userCanEditGeneral` determines whether or not we show an action, whereas I had to work harder to understand that from the nested ifs


```
  {% if exporterCreditRating %}
    {% set exporterCreditRatingValueHtml %}
      <span data-cy="exporter-table-credit-rating-value">{{ exporterCreditRating }}</span>
    {% endset %}
    {% set editExporterCreditRatingActionText = 'Change' %}
    {% set editExporterCreditRatingActionDataCyValue = 'exporter-table-change-credit-rating-link' %}
  {% else %}
    {% set exporterCreditRatingValueHtml %}
      {{ govukTag({
        text: 'Not added',
        classes: 'govuk-tag--yellow',
        attributes: {
          'data-cy': 'exporter-table-credit-rating-not-added'
        }
      }) }}
    {% endset %}
    {% set editExporterCreditRatingActionText = 'Add' %}
    {% set editExporterCreditRatingActionDataCyValue = 'add-credit-rating-link' %}
  {% endif %}
  
  {% if userCanEditGeneral %}
    {% set creditRatingRow = {
      key: { html: exporterCreditRatingHeadingHtml },
      value: { html: exporterCreditRatingValueHtml },
      actions: {
        items: [
          {
            href: "/case/" + caseId + "/underwriting/pricing-and-risk/edit",
            text: editExporterCreditRatingActionText,
            attributes: {
              'data-cy': editExporterCreditRatingActionDataCyValue
            }
          }
        ]
      }
    } %}  
  {% else %]
    {% set creditRatingRow = {
      key: { html: exporterCreditRatingHeadingHtml },
      value: { html: exporterCreditRatingValueHtml }
    } %}
  {% endif %}
```